ansible/roles/test/tasks/fdb.yml

```diff
@@ -153,6 +154,9 @@
     command: sonic-clear fdb all
```

Comment:
Please, execute "sonic-clear fdb all" in always block regardless of test status

Reply:
Good suggestion, thanks!